src/components/One/index.jsx

```diff
@@ -1,5 +1,30 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import "../../globals.css";
+
 function Index() {
-  return <div>Index</div>;
+  return (
+    <div className="flex flex-col items-center justify-center  text-white  bg-[#141619] min-h-[80px]  px-48 pt-[100px] pb-[128px] relative">
```

Comment:
avoid using abritary values if possible, refer docs for predefined taliwnd css values [refer this for padding](https://tailwindcss.com/docs/padding) (ctrl + k) to serach in docs